src/main/java/uk/ac/ebi/subs/filecontentvalidator/service/FileContentValidator.java

```diff
@@ -50,11 +53,19 @@ public SingleValidationResult validateFileContent() throws IOException, Interrup
         if (last >= 0) {
             resultMessage = output.substring(last, output.length());
             if (resultMessage.contains(ERROR_RESULT_BEGINNING)) {
-                validationError = resultMessage.replace(ERROR_RESULT_BEGINNING, "").trim();
+                validationErrors.add(resultMessage.replace(ERROR_RESULT_BEGINNING, "").trim());
```

Comment:
This should return a singleValidationError object with the error message you build here.